UdpHosts/GameServer/Entities/Character/Character.cs

```diff
@@ -38,9 +26,9 @@ public Character(IShard owner, ulong eid) : base(owner, eid)
         public Quaternion Rotation { get; set; }
         public Vector3 Velocity { get; set; }
         public Vector3 AimDirection { get; set; }
-        public CharMovement MovementState { get; set; }
+        internal MovementStateContainer MovementStateContainer { get; set; } = new();
```

Comment:
Might want to move it down below the public fields, given that it's internal now

Reply:
Resolved in #[3b38dac](https://github.com/themeldingwars/PIN/pull/16/commits/3b38dac1e11b9ef47dcd056f782b1363945bd3c4)